manifests/issue.pp

```diff
@@ -0,0 +1,34 @@
+# === Copyright
+#
+# Copyright 2014, Deutsche Telekom AG
```

Comment:
@bitvijays from my POV here should be your name, as the entire code initially is developed by you.

@chris-rock what is our view? Do we have any common agreement here?